### PR TITLE
allow splits through proposal offer

### PR DESCRIPTION
**Pull request by CJentzsch**

### CODE REVIEW

File: PFOffer.sol [hunk 1]
```diff
@@ -182,10 +182,12 @@ contract PFOffer {
     }
 
     function sign() {
+        var (recipient,,,votingDeadline,,) = client.proposals(proposalID);
         if (msg.sender != address(originalClient) // no good samaritans give us ether
             || msg.value != totalCosts    // no under/over payment
             || dateOfSignature != 0       // don't sign twice
-            || !wasApprovedBeforeDeadline)// fail if the voteStatusCheck was not done
+            || !wasApprovedBeforeDeadline // fail if the voteStatusCheck was not done
+			|| now < votingDeadline + 3 days)
```
Comment (LefterisJP) [inline PFOffer.sol]: Please use spaces not tabs

File: PFOffer.sol [hunk 2]
```diff
@@ -182,10 +182,12 @@ contract PFOffer {
     }
 
     function sign() {
+        var (recipient,,,votingDeadline,,) = client.proposals(proposalID);
```
Comment (LefterisJP) [inline PFOffer.sol]: `recipient` is not used so you can replace it with a `_`. Don't leave it empty, the contract won't compile then.

File: PFOffer.sol (hunk 1 above)
Comment (CJentzsch) [inline PFOffer.sol]: ops

File: PFOffer.sol [hunk 3]
```diff
@@ -253,7 +255,7 @@ contract PFOffer {
     // to be sure that YEA was able to succeed 48 hours before the deadline
     function checkVoteStatus() noEther {
         var (,,,votingDeadline,,,,,,yea,nay,) = client.proposals(proposalID);
-        uint quorum = (yea + nay) * 100 / client.totalSupply();
+        uint quorum = yea * 100 / client.totalSupply();
```
Comment (LefterisJP) [inline PFOffer.sol]: :+1:

**Pull request merged**